### PR TITLE
2 project initialization

### LINKED ISSUE #2: project-initialization

### PR DESCRIPTION
**Pull request by MaciejIwan**

### CODE REVIEW

**Review by matiduda — COMMENTED** [review]
The implementation looks good and it appears that the code is well organized and follows best practices. It seems that all relevant tests have been included and are passing as expected. Based on what has been presented in this pull request, it seems ready for review and merge.

**Review by matiduda — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "@ethersproject/abi": "^5.4.7",
+    "@ethersproject/providers": "^5.4.7",
+    "@nomicfoundation/hardhat-chai-matchers": "^1.0.0",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
+    "@nomicfoundation/hardhat-toolbox": "^1.0.1",
+    "@nomiclabs/hardhat-ethers": "^2.0.0",
+    "@nomiclabs/hardhat-etherscan": "^3.0.0",
+    "chai": "^4.2.0",
+    "ethers": "^5.4.7",
+    "hardhat": "^2.11.0",
```
Comment (ad-jusk) [inline package.json:29]: When I set up the project, hardhat version ^2.12.5 was installed. It's most likely we have different node versions installed. It shouldn't matter that much but let's make sure we are all using the same version.

**Review by MrDell1 — APPROVED** [review]

**Review by ad-jusk — APPROVED** [review]
Everything looks great, it's nice you included tests and made sure everything works correctly. We should deffinitely focus on using the same IDE and technologies during developing process to avoid unvanted conflicts and formatting issues.

**Review by MrDell1 — APPROVED** [review]
.

**Review by MrDell1 — APPROVED** [review]

**Review by MrDell1 — APPROVED** [review]

**Review by MrDell1 — APPROVED** [review]
Everything looks great, it's nice you included tests and made sure everything works correctly. We should deffinitely focus on using the same IDE and technologies during developing process to avoid unvanted conflicts and formatting issues.

**Comment by MaciejIwan** [comment]
> Everything looks great, it's nice you included tests and made sure everything works correctly. We should deffinitely focus on using the same IDE and technologies during developing process to avoid unvanted conflicts and formatting issues.

Yes, I agree, that's good idea. I would suggest using webstorm and webstorm default formatting style

**Review by MrDell1 — APPROVED** [review]

**Pull request merged**